index.html

```diff
@@ -10,18 +10,44 @@
   <body>
     <div class="container">
       <header>
-        <h1 class="logo">Riyana Abdi</h1>
-        <img src="SVG/Icon - Menu.svg" alt="" class="menuIcon" />
-        <ul class="menuList">
-          <li>Portfolio</li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#Contact">Contact</a></li>
-          <li>
-            <a href="mailto:karanjain2212@gmail.com"
+       <nav class="navbar">
+        <h1 class="logo">Riyana </h1>
+        <div class="humburger">
+          <img src="./img/Menu.png" alt="" id="openBtn">
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="nav-link">portofolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">contact</a>
+          </li>
+          <li class="nav-item">
+            <a href="mailto:Riyanagwl@gmail.com"
```

Comment:
- [ ] Clicking on mobile menu options suppose to take me to the correct part of the page, but that's not happening. So please fix that.

![chrome-capture-2022-9-17](https://user-images.githubusercontent.com/68796232/196244926-290b9c1f-4c7d-4552-86d5-4c79a3f6a846.gif)